grpc/core/pom.xml

```diff
@@ -26,13 +26,41 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>helidon-grpc-core</artifactId>
-    <name>Helidon gRPC related modules</name>
+    <name>Helidon gRPC Core</name>
 
     <dependencies>
         <dependency>
             <groupId>io.helidon.common</groupId>
             <artifactId>helidon-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.helidon.common</groupId>
+            <artifactId>helidon-common-config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.common</groupId>
+            <artifactId>helidon-common-configurable</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.config</groupId>
+            <artifactId>helidon-config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.config</groupId>
+            <artifactId>helidon-config-metadata</artifactId>
```

Comment:
This should never be a transitive dependency. It should be `<optional>true</optional>`

Reply:
There was a class removed in the other PR and not this one, all these deps go away.